### ISSUE TITLE
New package for inclusion - docmeteor

### ISSUE THREAD

**Issue by jankapunkt** [Meteor-Community-Packages/organization#54]
## Package/project name & description

It's an npm package, that allows to create API markdown files from JSDoc with Meteor specific extensions.

## Links
* [Github repository - https://github.com/raix/docmeteor](https://github.com/raix/docmeteor)

## Current status of the project

 * Abandoned

 ## Reasoning

The project allows an easy out-of-the-box API to Markdown creation and brings Meteor-specific extensions with it. Example output can be viewed currently in the [AutoForm API documentation](https://github.com/Meteor-Community-Packages/meteor-autoform/blob/devel/api.md)

It has not very widespread use but this could change. I (@jankapunkt) would be willing to support alongside of AutoForm.
 
 ## Progress
 - [ ] Discussion finished
 - [ ] Moved / Forked
 - [ ] Team Created
 - [ ] Meteor Org Created and set as maintainer
 - [ ] `communitypackages` org added as a maintainer on Atmosphere

**Comment by harryadel** [comment]
Looks like a hidden gem. If you're willing to maintain it I see nothing wrong with moving forward @copleykj

**Comment by jankapunkt** [comment]
@harryadel I see value in it, because it makes API documentation for packages very easy and this org is about packages so we may benefit a lot.

**Comment by copleykj** [comment]
@jankapunkt I don't have any objections. I'll let @StorytellerCZ weigh in if he would like.

**Comment by StorytellerCZ** [comment]
No objections here either. We can also utilize our npm org for this. We should also start applying it across packages.

**Comment by StorytellerCZ** [comment]
Alright who is going to take responsibility for this?

**Comment by jankapunkt** [comment]
I would go for it. Did someone of you already contacted @raix in the past? Should I simply open an issue or how can we manage that?

**Comment by copleykj** [comment]
@jankapunkt I would say start with filing an issue, and if no response in the next week or so, then maybe try other ways to contact him.

**Comment by harryadel** [comment]
I had once contacted him via linkedin but yeah I second @copleykj It's better to delay intruding on his personal space and open up an issue first.

**Comment by copleykj** [comment]
Yes, I personally don't mind any form of direct contact, but I know that many devs do, and it's best to respect that unless necessary or you know specifically that they don't mind.

**Comment by StorytellerCZ** [comment]
I would prefer if we could get at least access to the npm name. Moving the GitHub repo is not necessary in this case.
